src/store/plugins/presenceReporter.js

```diff
@@ -13,7 +13,7 @@ export default store => {
     .forEach(event => document.addEventListener(event, resetTimer))
 
   // listen on tab & window visibility, mostly works on desktop browser
-  Events.$on('app:visibility', state => {
+  /* Events.$on('app:visibility', state => {
```

Comment:
Could be replaced with http://quasar-framework.org/components/app-visibility.html

Reply:
... but need to check if it also works without the quasar plugin